app/views/content/landing/is-teaching-right-for-me/_facts_and_figures.html.erb

```diff
@@ -0,0 +1,30 @@
+<article>
+  <section class="container">
+      <h2 class="heading--box-blue">Facts and figures</h2>
+
+      <div class="inset">
+        <%= render Content::FactListComponent.new(
+          [
+            {
+              icon: "icon-money",
+              value: "£28k",
+              title: "minimum starting salary",
+              link: link_to("Salaries and benefits", page_path("salaries-and-benefits")),
+            },
+            {
+              icon: "icon-money",
```

Comment:
@ethax-ross @gazcarr can we change this icon?

Reply:
![image](https://user-images.githubusercontent.com/402788/201890922-0b1b7da6-6d06-4145-91c3-3bcfeef0ba43.png)

These were the icons I suggested, I am not sure what the names are for these though.. @ethax-ross ?